test/test_torch.py

```diff
@@ -9908,6 +9908,24 @@ def check2d(x, y):
                 y[0][1] = 3
                 self.assertTrue(x[0][1] == 3)
 
+    @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
+    def test_to_numpy_bool(self):
+        x = torch.tensor([True, False], dtype=torch.bool)
+        self.assertEqual(x.dtype, torch.bool)
+
+        y = x.numpy()
+        self.assertEqual(y.dtype, np.bool)
+        for i in range(len(x)):
+            self.assertEqual(x[i], y[i])
+
+        # test for a scalar case
+        x = torch.tensor([True], dtype=torch.bool)
```

Comment:
this isn't a scalar, `torch.tensor(True, ...)` would be a scalar.

Reply:
I thought you were going to change this to torch.tensor(True, dtype=torch.bool) ?